Gemfile

```diff
@@ -2,22 +2,22 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.1.10'
+gem 'rails', '~> 5'
```

Comment:
@jwpincus honestly, i think it's probably fine. as long as you weren't doing anything that was specific to rails 4, which i assume you weren't doing anything _that_ complicated at this point, you should be fine 👍 

i agree though - probably not the best idea after you've started working on code.

do you need to use rails 5 for something specific?